### PR TITLE
scale firefox and chrome

### PR DESCRIPTION
**Pull request by jburel**
Increase number of firefox and chrome

Related to https://github.com/openmicroscopy/devspace/pull/88

### CODE REVIEW

File: tasks/devspace-runtime.yml [hunk 1]
```diff
@@ -26,5 +29,11 @@
     - "nginxjenkins.devspace_nginxjenkins_1.state.running"
     - "redis.devspace_redis_1.state.running"
     - "seleniumhub.devspace_seleniumhub_1.state.running"
-    - "seleniumfirefox.devspace_seleniumfirefox_1.state.running"
-    - "seleniumchrome.devspace_seleniumchrome_1.state.running"
+    - "seleniumfirefox.devspace_seleniumfirefox_{{item}}.state.running"
+    - "seleniumchrome.devspace_seleniumchrome_{{item}}.state.running"
+  with_items:
```
Comment (joshmoore) [inline tasks/devspace-runtime.yml]: You might want to try http://docs.ansible.com/ansible/latest/playbooks_loops.html#looping-over-integer-sequences (and perhaps the `5` here could be a variable)

**Comment by jburel** [comment]
@joshmoore thanks for the tip.
Done

**Comment by jburel** [comment]
I have tested the change by re-provisioning the devspace
everything is again up and running https://10.0.51.135:32784/

**Comment by joshmoore** [comment]
LGTM.

**Pull request merged**